src/neorados/RADOSImpl.h

```diff
@@ -47,6 +48,7 @@ class RADOS : public Dispatcher
 
   boost::asio::io_context& ioctx;
   boost::intrusive_ptr<CephContext> cct;
+  tracing::Tracer tracer;
```

Comment:
why do we add a new tracer to the neorados client?
won't that create an issue if there is already a tracer in the app using the client?